examples/AzureSDKDemoSwift/Source/BlobDownloadViewController.swift

```diff
@@ -203,8 +203,17 @@ extension BlobDownloadViewController: UITableViewDelegate, UITableViewDataSource
                 blob: blobName,
                 fromContainer: containerName,
                 toFile: destination,
-                withOptions: options
-            )
+                withOptions: options,
+                progressHandler: { transfer in
+                    if transfer.transferType == .download {
+                        tableView.reloadData()
+                    }
+                }
+            ) { transfer in
+                if transfer.transferType == .download {
+                    tableView.reloadData()
+                }
+            }
```

Comment:
This is an idiomatic sample of what this usage would look like based on existing usage in SwiftUI. You could of course also put the `completionHandler` as a parameter in the method call.